amazon-eks-nodegroup.yaml

```diff
@@ -121,6 +129,24 @@ Parameters:
       - c5n.4xlarge
       - c5n.9xlarge
       - c5n.18xlarge
+      - c6g.medium
```

Comment:
Instance types look good and match [files/eni-max-pods.txt](https://github.com/mogren/amazon-eks-ami/blob/master/files/eni-max-pods.txt)